Gemfile.lock

```diff
@@ -1,95 +1,93 @@
 PATH
   remote: .
   specs:
-    lockstep_rails (0.3.45)
+    lockstep_rails (0.3.46)
       rails
 
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.0.3.1)
-      actionpack (= 7.0.3.1)
-      activesupport (= 7.0.3.1)
+    actioncable (6.1.7)
+      actionpack (= 6.1.7)
+      activesupport (= 6.1.7)
```

Comment:
bro why is rails changed from 7.0.3.1 to 6.1.7 :/

Reply:
It asked me to run bundle update and this happened. Not sure tho ! Will rectify

Reply:
Fixed it. It was an issue w my ruby version so it made me update bundle i think 😓